src/Reducer/cartReducer.jsx

```diff
@@ -0,0 +1,22 @@
+export function cartReducer(state, action) {
+    switch(action.type) {
+
+        case "ADD_TO_CART":
+            return { ...state, cart: [...state.cart, { ...action.payload, qty: 1 }] 
+        };
+      
+        case "REMOVE_FROM_CART":
+        return {
+            ...state,
+            cart: [...state.cart.filter((item) => item.id !== action.payload.id)]
+        };
+
+        case "MOVE_TO_CART":
```

Comment:
suggestion : you can eleminate this case by using addtocart case for move to cart functionality